src/Grid/Grid.js

```diff
@@ -135,17 +134,83 @@ class Grid extends React.Component {
         popup.document.close();
       });
   }
+  
+  exportTable = filtered => {
```

Comment:
@kadosh do we need this method in the same class?

Reply:
Maybe we will move that one and printTable as well to an API class (something similar to an angular service).

Reply:
Yeah, let's keep this class for now, and refactor it with typescript!